dotnet/samples/KernelSyntaxExamples/Example42_KernelBuilder.cs

```diff
@@ -80,8 +80,8 @@ public static Task RunAsync()
         using var memory = new SemanticTextMemory(memoryStorage, textEmbeddingGenerator);
         var skills = new SkillCollection();
         var templateEngine = new PromptTemplateEngine(loggerFactory);
-        var kernelConfig = new KernelConfig();
-        using var httpHandler = kernelConfig.HttpHandlerFactory.Create(loggerFactory);
+        var httpHandlerFactory = NullHttpHandlerFactory.Instance;
```

Comment:
This doesn't need to be done, does it? Is this why it was made public? Maybe we keep it internal and use the Basic or Polly factory here instead.